### PR TITLE
@mzikherman => [FIX artistInfo]: Fix missing section, add tests

### PR DESCRIPTION
**Pull request by damassi**
Fixes issue where content was missing and additionally adds some tests using the new test scaffold setup: 

screen shot 2018-10-09 at 3 37 00 pm

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 1c5ae28579030e6556225387b62f31a3eab8a3bb

https://deploy-preview-1385--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/ArtistInfo.tsx [hunk 1]
```diff
@@ -22,7 +22,12 @@ interface ArtistInfoProps {
   mediator?: Mediator
 }
 
+const Container = ({ children }) => (
+  <StackableBorderBox p={2}>{children}</StackableBorderBox>
+)
+
 export const ArtistInfo: SFC<ArtistInfoProps> = props => {
+  const showArtistBio = !!props.artist.biography_blurb.text
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/ArtistInfo.tsx]: 👍 nice

File: src/Styleguide/Components/MarketInsights.tsx [hunk 2]
```diff
@@ -100,7 +98,15 @@ export class MarketInsights extends React.Component<MarketInsightsProps> {
       return null
     }
 
-    const Container = this.props.border ? BorderBox : Box
+    let Container
+
+    if (this.props.Container) {
```
Comment (mzikherman) [inline src/Styleguide/Components/MarketInsights.tsx]: interesting approach to pass this around...makes sense from the layout/logic perspective

File: src/Apps/Artwork/Components/__tests__/ArtistInfo.test.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { renderRelayTree } from "DevTools/MockRelayRenderer"
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/__tests__/ArtistInfo.test.tsx]: lovely tests

**Pull request merged**